### DISCUSSION TITLE
Add Surrogate Index module

### DISCUSSION THREAD

**Discussion by paullo0106** [uber/causalml#418] [General]
@huigangchen introduced [The Surrogate Index: Combining Short-Term Proxies to Estimate Long-Term Treatment Effects More Rapidly and Precisely](https://www.nber.org/papers/w26463) and it's now on CausalML's roadmap. I'm creating a discussion thread to brainstorm further and track the development. 

- [x] Paper study group: hosted and done by @huigangchen  (internal doc [here](https://docs.google.com/document/d/1MyZ2wG10t43AP211i9x87TQK4d4FqbR8Lm9HNG88Ceg/edit))
- [x] Resource collection - read the [code repo](https://github.com/OpportunityInsights/Surrogates-Replication-Code) (in Stata) by JamesStratton 
- [x] Create python notebook prototype for Treatment effect estimation (Part A & B) - done by @huigangchen  
- [ ] Create python notebook prototype for Treatment effect estimation (Part C)
- [ ] Create python notebook prototype for Treatment effect estimation (Part D)
- [ ] Create python notebook prototype for Treatment effect estimation (Part E)
- [ ] Model and interface design (e.g. code structure, function input and output, and etc.)
- [ ] Model implementation
- [ ] Unit test
- [ ] Docs
- [ ] Notebook example

**Comment by paullo0106** [comment]
@huigangchen @ppstacy @vincewu51 Please feel free to add/ edit!

**Comment by ludovico-lanni** [comment]
Hello! Any news on this implementation plan?

**Comment by jeongyoonlee** [comment]
@huigangchen, Can you share the prototype notebook you implemented here?
